docs/manuals/core/config/data-node-config.md

```diff
@@ -41,6 +41,10 @@ The attributes available on data nodes are:
     storage type. It is copied in the dictionary properties of all the data nodes
     instantiated from this data node configuration.<br/>
 
+!!! Warning
```

Comment:
The syntax we use is: 
`!!! warning "title"`

Here, this could be:
`!!! warning "Reserved key"`

Reply:
Noted thanks!! I'll add that immediately 😃